### DISCUSSION TITLE
Exports monetary value

### DISCUSSION THREAD

**Discussion by GH2user** [hif2k1/battery_sim#128] [Q&A] [answered]
I love the integration and after a bit of struggles have it now running.
I do notice that the monetary value of my exports is listed as negative therefore skewing the total value.
My input for both prices is 0.3 euro/kWh for each,

Should I add the electricity price as negative or am I just misunderstanding the concept?

**Comment by hif2k1** [comment, marked as answer]
That is the expected behaviour.

Normally you would get paid for exporting electricity. If you generated
10kwh and exported it then you get paid for that. If instead you store it
in the battery you won't get that money which is why it is negative. You
will use that electricity later though and save importing much more
expensive energy which is why batteries save you money overall. Initially
on the batteries first charge though the net benefit will be negative until
you've used some energy from the battery which can be disconcerting. If you
don't get paid for exporting then you should set the tariff for the export
sensor to 0.

Say you get paid $0.10 for each kWh you export. During the day the sun
shines and you generate 10kwh. Without the battery you would have exported
that and earned $1. When the sun sets you're down $1 (negative). However,
in the evening you use that 10kwh in the house. If your tariff for
importing is $0.25 per kwh it would have cost you $2.50, but instead you
use the energy from the battery saving you that money. Overall at the end
of the day you've saved $1.50 by having the battery.
